src/app/components/DataSelection/DataSelectionFilters.tsx

```diff
@@ -132,7 +132,7 @@ const DataSelectionFilters: FunctionComponent<Props> = ({
                           }
                         >
                           <span className="c-data-selection-available-filters__item-label">
-                            <span className="qa-option-label">{option.label}</span>
+                            <span className="qa-option-label">{option.label || '(geen)'}</span>
```

Comment:
Would also be possible to use the `??` operator here.

```suggestion
                            <span className="qa-option-label">{option.label ?? '(geen)'}</span>
```